samples/nrf9160/modem_shell/prj.conf

```diff
@@ -128,6 +133,8 @@ CONFIG_MULTICELL_LOCATION_MAX_NEIGHBORS=17
 
 # Cellular positioning services
 CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
+# Uncomment the following if CONFIG_NRF_CLOUD_MQTT is enabled
```

Comment:
Somehow, cannot do that just in this PR, we should eventually make these configurations easier between MQTT and REST.

At here, maybe worth to note that basically it is either MQTT or REST for the nrf cloud services. 

Also was thinking if CONFIG_NRF_CLOUD_CELL_POS should have CONFIG_NRF_CLOUD_MQTT as a dependency, but that is not a subject of this PR.

Reply:
I totally agree, MQTT / REST configuration is a mess now.

I'll improve the comment.

Yes, CONFIG_NRF_CLOUD_CELL_POS should have CONFIG_NRF_CLOUD_MQTT as a dependency. It won't even compile if  CONFIG_NRF_CLOUD_CELL_POS  is enabled and CONFIG_NRF_CLOUD_MQTT is not.

Reply:
Maybe @jhirsi could close this as dependencies are handled with:
https://github.com/nrfconnect/sdk-nrf/pull/6223
https://github.com/nrfconnect/sdk-nrf/pull/6244

Reply:
Excellent, resolving.